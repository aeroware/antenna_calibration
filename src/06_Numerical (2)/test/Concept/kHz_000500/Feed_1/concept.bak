1.1   Testcube - V0 mesh                                                                                                                                        2.1       0                                                                                                                                                     2.2       2     0     8     0                                                                                                                                   2.3      0.0000     0.0000 0.5000         0.0000     0.0000     1.5000               1.0000    3                                                                2.3      0.0000 0.5000         0.0000     0.0000     1.5000     0.0000               1.0000    3                                                                2.6       1                                                                                                                                                     3.1       1                                                                                                                                                     3.2       1                                                                                                                                                     3.3  m1       0.000                                                                                                                                             4.1       1                                                                                                                                                     4.2  0.50000000                                                                                                                                                 5.1       1                                                                                                                                                     5.2       1     0                                                                                                                                               5.3   0.5000E+08                                                                                                                                                5.4          2                                                                                                                                                  6.1       0                                                                                                                                                     7.1       0                                                                                                                                                     7.3       0                                                                                                                                                     9.0       0                                                                                                                                                     9.1       0                                                                                                                                                     10.3      0                                                                                                                                                     11.1      1                                                                                                                                                     11.10     1                                                                                                                                                     11.11     0                                                                                                                                                     12.1     12                                                                                                                                                     