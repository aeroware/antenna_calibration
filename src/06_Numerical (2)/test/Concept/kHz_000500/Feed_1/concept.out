 *
 * This is CONCEPT back end, Version 10.0
 *       Single Processor Version
 * (c) Copyright 1984-2011 TUHH E-18
 *
  
  
   The current data file  
   =====================
  
1.1  characterization of the structure                                          
 Testcube - V0 mesh                                                             
2.1   symmetry with respect to the xz plane, yz plane or both                   
     0                                                                          
2.2   number of wires, id. ground, segments per wavelength, coating             
     2     0     8     0                                                        
2.30  wire 1   : coordinates                                                    
    0.0000     0.0000 0.5000         0.0000     0.0000     1.5000               
2.31  wire 1   : radius, no. of basis functions, RB, EP, MY                     
    1.0000    3                                                                 
2.30  wire 2   : coordinates                                                    
    0.0000 0.5000         0.0000     0.0000     1.5000     0.0000               
2.31  wire 2   : radius, no. of basis functions, RB, EP, MY                     
    1.0000    3                                                                 
2.6   check wire grid                                                           
     1                                                                          
3.1  print of currents                                                          
     1                                                                          
3.2  excitation                                                                 
     1                                                                          
3.3  generator position, internal resistance (excitation=1)                     
m1       0.000                                                                  
4.1  representation of frequency(ies) (1-6)                                     
     1                                                                          
4.2  basic frequency                                                            
0.50000000                                                                      
5.1  skin effect (1/0)                                                          
     1                                                                          
5.2  number: conductivity value(s), observation point(s) E tang                 
     1     0                                                                    
5.3  conductivity value(s) in S/m (max. 3 per line)                             
 0.5000E+08                                                                     
5.4  conductivity domain(s)                                                     
        2                                                                       
6.1  number of load impedances                                                  
     0                                                                          
7.1  current distribution on wires                                              
     0                                                                          
7.3  number of currents at discrete points                                      
     0                                                                          
9.0  rcs (0/1)                                                                  
     0                                                                          
9.1  number of field points                                                     
     0                                                                          
10.3  total number of "cable wires"                                             
     0                                                                          
11.1  surface currents (0/1/2/3/4)                                              
     1                                                                          
11.10 check surface grid (0/1)                                                  
     1                                                                          
11.11 physical optics (0/1)                                                     
     0                                                                          
12.1  direct:11,12,13 spec. direct solv.:21,22 iter. solver:31,32,33 aca/svd:41 
    12                                                                          
  
  
  
  Wire types computed by the program: 
  ===========================================
  
  wire number:       1 ,  identification:      15
  wire number:       2 ,  identification:      15
  
  
  
 Wire data
 =========
  
 Connections at the bottom with other and image wires
 -> 0 will be printed.
 Pos. direction of current flow away from node point (kinks).
 -> neg. wire number
 Pos. direction of current flow towards the node point 
 -> pos. wire number
 Connection with surface patch
 -> 55000+number of surface patch is printed
 Wire number:            1
 
 ---------------------------------------------------------------------
      Xa          Ya           Za         Xe          Ye           Ze
     0.000       0.000      0.5000       0.000       0.000       1.500    
 Radius (mm):  1.000       ; no. of basisf. :   3   ; length (m):   1.000    
 Wire number(s) of the connection(s) at the beginning :     10000013
 No connection at the end
  
 Wire number:            2
 
 ---------------------------------------------------------------------
      Xa          Ya           Za         Xe          Ye           Ze
     0.000      0.5000       0.000       0.000       1.500       0.000    
 Radius (mm):  1.000       ; no. of basisf. :   3   ; length (m):   1.000    
 Wire number(s) of the connection(s) at the beginning :     10000001
 No connection at the end
  
 

  Frequency of step:     1           0.50000E+00 MHz; 
  Total number of wire current amplitudes:            6
  Total number of patch current amplitudes:           44
  Total number of unknowns:           50
  
  Generator at the middle of wire   1
  ------------------------------------
  
                          Real part    imag. part      magnitude     phase (degrees)
  Input impedance (Ohm): 0.20154E-01 -0.68524E+05  0.68524E+05 -0.90000E+02

  Internal resist.:   0.0000000E+00  +j   0.0000000E+00  Ohm
  
  Input power:   2.1461153E-12  Watt
  
 
  Current distribution on   1. wire in A (  3 current functions)
  ------------------------------------------------------------
    Real part       imag. part      magnitude     phase (degees)  wire coord. (m)   
  0.51378E-11     0.13701E-04     0.13701E-04     0.90000E+02   0.25000E+00
  0.42922E-11     0.14593E-04     0.14593E-04     0.90000E+02   0.50000E+00
  0.24674E-11     0.72363E-05     0.72363E-05     0.90000E+02   0.75000E+00
 
  Current distribution on   2. wire in A (  3 current functions)
  ------------------------------------------------------------
    Real part       imag. part      magnitude     phase (degees)  wire coord. (m)   
 -0.27271E-12    -0.40295E-06     0.40295E-06    -0.90000E+02   0.25000E+00
 -0.21397E-12    -0.30730E-06     0.30730E-06    -0.90000E+02   0.50000E+00
 -0.12095E-12    -0.17138E-06     0.17138E-06    -0.90000E+02   0.75000E+00
  
  
                          Surface currents (A/m)
                          ----------------------
        Element no.       real part       imag. part     magnitude     phase (degrees)
  
                          Space number:            0
  
     1    25 =>    13   0.23683E-11     -0.34403E-05    0.34403E-05   -0.90000E+02 Current in A  
     2    25 =>    14  -0.58933E-11     -0.33059E-05    0.33059E-05   -0.90000E+02 Current in A  
     3    25 =>    15   0.23683E-11     -0.34403E-05    0.34403E-05   -0.90000E+02 Current in A  
     4    25 =>    16  -0.39181E-11     -0.33401E-05    0.33401E-05   -0.90000E+02 Current in A  
     5    26 =>     1  -0.69899E-11     -0.30036E-06    0.30036E-06   -0.90001E+02 Current in A  
     6    26 =>     2   0.92675E-11      0.69031E-06    0.69031E-06    0.89999E+02 Current in A  
     7    26 =>     3  -0.11264E-10     -0.64902E-06    0.64902E-06   -0.90001E+02 Current in A  
     8    26 =>     4   0.92675E-11      0.69031E-06    0.69031E-06    0.89999E+02 Current in A  
     9    24 =>    23   0.10148E-10      0.10678E-05    0.10678E-05    0.89999E+02               
    10    24 =>     2  -0.10744E-10      0.42264E-07    0.42264E-07    0.90015E+02               
    11    24 =>    21   0.52581E-11     -0.53361E-06    0.53361E-06   -0.89999E+02               
    12    23 =>    15   0.86421E-11      0.21207E-05    0.21207E-05    0.90000E+02               
    13    23 =>    22  -0.22228E-11     -0.11101E-05    0.11101E-05   -0.90000E+02               
    14    22 =>    20  -0.83260E-12      0.15957E-07    0.15957E-07    0.90003E+02               
    15    22 =>    21  -0.85163E-12     -0.53129E-06    0.53129E-06   -0.90000E+02               
    16    21 =>     6   0.35394E-11     -0.39191E-06    0.39191E-06   -0.89999E+02               
    17    20 =>    19  -0.65307E-13      0.11353E-05    0.11353E-05    0.90000E+02               
    18    20 =>    17  -0.92572E-12     -0.51041E-06    0.51041E-06   -0.90000E+02               
    19    19 =>    16  -0.21126E-12      0.21885E-05    0.21885E-05    0.90000E+02               
    20    19 =>    18   0.64724E-13     -0.11353E-05    0.11353E-05   -0.90000E+02               
    21    18 =>     9   0.83266E-12     -0.15957E-07    0.15957E-07   -0.89997E+02               
    22    18 =>    17  -0.92612E-12     -0.51041E-06    0.51041E-06   -0.90000E+02               
    23    17 =>     5  -0.89966E-12     -0.35848E-06    0.35848E-06   -0.90000E+02               
    24    16 =>    13  -0.29428E-11      0.23961E-07    0.23961E-07    0.90007E+02               
    25    16 =>    15  -0.29429E-11      0.23961E-07    0.23961E-07    0.90007E+02               
    26    15 =>    14   0.12595E-10     -0.16781E-06    0.16781E-06   -0.89996E+02               
    27    14 =>     3   0.11899E-10     -0.23628E-05    0.23628E-05   -0.90000E+02               
    28    14 =>    13  -0.12595E-10      0.16781E-06    0.16781E-06    0.90004E+02               
    29    13 =>    12  -0.86422E-11     -0.21207E-05    0.21207E-05   -0.90000E+02               
    30    12 =>     9  -0.22225E-11     -0.11101E-05    0.11101E-05   -0.90000E+02               
    31    12 =>    11  -0.10149E-10     -0.10678E-05    0.10678E-05   -0.90001E+02               
    32    11 =>     4  -0.10744E-10      0.42264E-07    0.42264E-07    0.90015E+02               
    33    11 =>    10   0.52577E-11     -0.53361E-06    0.53361E-06   -0.89999E+02               
    34    10 =>     8   0.35394E-11     -0.39191E-06    0.39191E-06   -0.89999E+02               
    35    10 =>     9   0.85133E-12      0.53129E-06    0.53129E-06    0.90000E+02               
    36     8 =>     7   0.57249E-11     -0.88263E-07    0.88263E-07   -0.89996E+02               
    37     8 =>     5   0.18827E-12      0.12908E-07    0.12908E-07    0.89999E+02               
    38     7 =>     6  -0.57251E-11      0.88263E-07    0.88263E-07    0.90004E+02               
    39     7 =>     1   0.87429E-11      0.21017E-06    0.21017E-06    0.89998E+02               
    40     6 =>     5   0.18820E-12      0.12908E-07    0.12908E-07    0.89999E+02               
    41     4 =>     1  -0.15310E-11     -0.16467E-06    0.16467E-06   -0.90001E+02               
    42     4 =>     3  -0.36665E-12      0.17460E-05    0.17460E-05    0.90000E+02               
    43     3 =>     2   0.36601E-12     -0.17460E-05    0.17460E-05   -0.90000E+02               
    44     2 =>     1  -0.15315E-11     -0.16467E-06    0.16467E-06   -0.90001E+02               
  
  
                         Normal component of E
                         ---------------------
    Number    real part      imag. part      magnitude     phase (degrees)
  
                         Space number:            0
  
     1     -0.46458E-01     0.59308E-07     0.46458E-01     0.18000E+03
     2     -0.55446E-01     0.19354E-07     0.55446E-01     0.18000E+03
     3     -0.78032E-01    -0.16776E-07     0.78032E-01    -0.18000E+03
     4     -0.55446E-01     0.19317E-07     0.55446E-01     0.18000E+03
     5     -0.48924E-01     0.91092E-07     0.48924E-01     0.18000E+03
     6     -0.48695E-01     0.92304E-07     0.48695E-01     0.18000E+03
     7     -0.48172E-01     0.92970E-07     0.48172E-01     0.18000E+03
     8     -0.48695E-01     0.92304E-07     0.48695E-01     0.18000E+03
     9     -0.61146E-01     0.19687E-07     0.61146E-01     0.18000E+03
    10     -0.51925E-01     0.60913E-07     0.51925E-01     0.18000E+03
    11     -0.60397E-01     0.21616E-07     0.60397E-01     0.18000E+03
    12     -0.83500E-01    -0.15184E-07     0.83500E-01    -0.18000E+03
    13     -0.17026E+00    -0.33703E-08     0.17026E+00    -0.18000E+03
    14     -0.16975E+00    -0.28557E-08     0.16975E+00    -0.18000E+03
    15     -0.17026E+00    -0.33696E-08     0.17026E+00    -0.18000E+03
    16     -0.17048E+00    -0.46689E-08     0.17048E+00    -0.18000E+03
    17     -0.52250E-01     0.58927E-07     0.52250E-01     0.18000E+03
    18     -0.61244E-01     0.18986E-07     0.61244E-01     0.18000E+03
    19     -0.83826E-01    -0.17157E-07     0.83826E-01    -0.18000E+03
    20     -0.61244E-01     0.18958E-07     0.61244E-01     0.18000E+03
    21     -0.51925E-01     0.60917E-07     0.51925E-01     0.18000E+03
    22     -0.61146E-01     0.19698E-07     0.61146E-01     0.18000E+03
    23     -0.83500E-01    -0.15180E-07     0.83500E-01    -0.18000E+03
    24     -0.60397E-01     0.21579E-07     0.60397E-01     0.18000E+03
 
 CONCEPT Timings:

 Setup equation system (s)...................            0
 Solve equation system (s)...................            0
 Cleanup (s).................................            0
 
 CONCEPT terminated normally.

